### ISSUE TITLE
Default ConfigMap getting deleted in Controller's namespace

### ISSUE THREAD

**Issue by jace-ys** [tektoncd/triggers#523]
# Expected Behavior

Deleting all EventListeners in a namespace should not delete the logging ConfigMap that the `tekton-triggers-controller` and `tekton-triggers-webhook` rely on.

# Actual Behavior

When running EventListener's in the same namespace as `tekton-triggers-controller`, deleting all EventListeners in the namespace causes the default ConfigMap `config-logging-triggers` (created as part of the [installation process](https://github.com/tektoncd/triggers/blob/master/docs/install.md#installing-tekton-triggers-1)) to be pruned - a feature of this PR: https://github.com/tektoncd/triggers/pull/250. As the `tekton-triggers-webhook` relies on this ConfigMap, the webhook pod is not able to come up properly, and you can no longer submit Triggers resources. You will need to recreate that ConfigMap to make the pod available again.

A related question: should I not be running Tekton custom resources in the same namespace as the controllers?

# Steps to Reproduce the Problem

1. Install Tekton Triggers into the `tekton-pipelines` namespace
2. Install an EventListener into the same namespace
3. Delete the EventListener
4. See the `config-logging-triggers` no longer existing: `kubectl --namespace tekton-pipelines get configmap`

# Additional Info

Kubernetes version:

- Output of kubectl version:

```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.4", GitCommit:"8d8aa39598534325ad77120c120a22b3a990b5ea", GitTreeState:"clean", BuildDate:"2020-03-12T23:40:44Z", GoVersion:"go1.14", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15+", GitVersion:"v1.15.9-gke.26", GitCommit:"525ce678faa2b28483fa9569757a61f92b7b0988", GitTreeState:"clean", BuildDate:"2020-03-06T18:47:39Z", GoVersion:"go1.12.12b4", Compiler:"gc", Platform:"linux/amd64"}
```

- Tekton Pipeline version:

```
Client version: 0.8.0
Pipeline version: v0.11.0
```

**Comment by dibyom** [comment]
Interesting! Definitely a bug. To answer your question, usually we do not create the Trigger resources in the same namespace as the controllers which would probably explain why we have not run into this. Regardless, even if someone does use the same namespace, we should not delete the config-map.

/kind bug

**Comment by dibyom** [comment]
Fixed by #530 
/close
